src/index.html

```diff
@@ -1,14 +1,37 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="navigation">
+      <div class="navigation__wraper">
+        <div class="navigation__logo">
+          <img src="./images/logo.png" alt="logo">
+        </div>
+        <nav class="navigation__nav">
+          <ul class="navigation__list">
+            <li class="navigation__item"><a href="#" class="navigation__link is-active">Apple</a></li>
+            <li class="navigation__item"><a href="#" class="navigation__link">Samsung</a></li>
+            <li class="navigation__item"><a href="#" class="navigation__link">Smartphones </a></li>
+            <li class="navigation__item">
+              <a href="#" class="navigation__link"
+                 data-qa="hover">Laptops & Computers</a>
```

Comment:
```suggestion
              <a 
                href="#" 
                class="navigation__link"
                data-qa="hover"
              >
                Laptops & Computers
              </a>
```
![image](https://github.com/mate-academy/layout_moyo-header/assets/96409563/581b0e17-7945-4eb5-b7d5-b0cc5fb93676)